src/logic/embedding.lean

```diff
@@ -33,6 +33,13 @@ end function
 protected def equiv.to_embedding {α : Sort u} {β : Sort v} (f : α ≃ β) : α ↪ β :=
 ⟨f, f.injective⟩
 
+instance equiv.coe_embedding {α : Sort u} {β : Sort v} : has_coe (equiv α β) (α ↪ β) :=
+⟨equiv.to_embedding⟩
+
+instance equiv.perm.coe_embedding {α : Sort u} : has_coe (equiv.perm α) (α ↪ α) :=
+⟨equiv.to_embedding⟩
```

Comment:
I would have expected this second one to not be needed (because `perm` is `@[reducible]`) -- but the test file I added failed without adding both instances. Does reducible not do what I think it does in this situation?

Reply:
I don't have an answer to your question (yet), but this might be slightly better as
```suggestion
equiv.coe_embedding
```